src/profile/components/Settings/Apps/AppConnections.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useContext, useEffect } from 'react';
+
+import { Message } from '@dotkomonline/design-system';
+import { IUserContext, UserContext } from 'authentication/providers/UserProvider';
+import useAsync from 'common/hooks/useAsync';
+import { getUserConsent } from 'oidc_clients/api/user_consent';
+import { revokeuserConsent } from 'oidc_clients/api/user_consent';
+import OidcClient from 'oidc_clients/components/OidcClient/index';
+import { IOidcClient } from 'oidc_clients/models/client';
+import { IUserConsent } from 'oidc_clients/models/user_consent';
+
+import style from './apps.less';
+
+interface IRemoveState {
+  client_id: number;
+  client?: IOidcClient;
+}
+
+export const AppConnections = () => {
+  const userContext = useContext<IUserContext>(UserContext);
+  const user = userContext.user;
+
+  if (!user) {
+    return <></>;
+  }
+
+  const [removeConsentState, setRemoveConsentState] = React.useState<IRemoveState>({
```

Comment:
Would go for importing `useState` since that is done all other places in the project

Reply:
:+1: